docs/application/native/guides/app-management/sticker.md

```diff
@@ -0,0 +1,431 @@
+
+    return sticker_data;
+}
+
+void
+insert_sticker_data()
+{
+    sticker_data_h data_handle;
+    int ret;
+
+    /* Your code */
+
+    data_handle = set_sticker_data(type, uri, keyword, len, group, thumbnail, description);
+
+    /* Your code */
+
+    /* Destroys a sticker data handle */
+    ret = sticker_data_destroy(data_handle);
+    if (ret != STICKER_ERROR_NONE)
+        /* Error handling */
+}
+```
+
+> **Note**
+>
+> When the Sticker data handle is no longer needed, destroy it.
+
+<a name="set_json"></a>
+### Setting the sticker information in the JSON file
+
+The following JSON file shows how to set the sticker information to be stored into the Sticker database:
+
+```
+{
+    "sticker":
+    [
+        {
+            "type" : 1,  /* Sets the URI type of the sticker */
+            "uri" : "/res/face/slightly_smiling_face.png",  /* Sets the URI of the sticker */
+            "keyword" : ["face", "smile"],  /* Adds a keyword of the sticker to the list */
+            "group" : "face",  /* Sets the group name of the sticker */
+            "thumbnail" : "/res/thumbnail/face/slightly_smiling_face.png",  /* Sets the thumbnail local path of the sticker */
+            "description" : "A yellow face with simple, open eyes and a thin, closed smile."  /* Sets the description of the sticker */
+        },
+        {
+            "type" : 1,
+            "uri" : "/res/face/heart_eyes.png",
+            "keyword" : ["face", "love", "adoration"],
+            "group" : "face",
+            "thumbnail" : "/res/thumbnail/face/heart_eyes.png",
+            "description" : "A yellow face with an open smile, sometimes showing teeth, and red, cartoon-styled hearts for eyes."
+        },
+        ...
+        {
+            ...
+        }
+    ]
+}
+```
+
+<a name="insert_sticker"></a>
+### Inserting the sticker information
+
+You can use the following methods to store sticker information into the Sticker database:
+
+- To store one sticker information, use the `sticker_provider_insert_data()` function:
+
+  ```
+  void
+  insert_sticker_data(sticker_provider_h sticker_provider, sticker_data_h sticker_data)
+  {
+      int ret;
+      ret = sticker_provider_insert_data(sticker_provider, sticker_data);
+      if (ret != STICKER_ERROR_NONE)
+          /* Error handling */
+  }
+  ```
+
+- To store multiple sticker information, use the `sticker_provider_insert_data_by_json_file()` function:
+
+  ```
+  void
+  insert_finished_cb(sticker_error_e error, void *user_data)
+  {
+      /* Your code */
+  }
+
+  void
+  insert_sticker_data(sticker_provider_h sticker_provider, const char* json_path)
+  {
+      int ret;
+      ret = sticker_provider_insert_data_by_json_file(sticker_provider, json_path, insert_finished_cb, NULL);
+      if (ret != STICKER_ERROR_NONE)
+          /* Error handling */
+  }
+  ```
+
+- To store sticker information without using the above functions, add the following metadata to the `tizen-manifest.xml` file:
```

Comment:
How about changing sample code of line 214 and description of line 212?


 To store sticker information without using `sticker_provider_insert_data()` and `sticker_provider_insert_data_by_json_file()`, add the following metadata to the `tizen-manifest.xml` file:

  ```
  <category name="http://tizen.org/category/sticker"/>
  <!-- Sets the JSON local path -->
  <metadata key="http://tizen.org/metadata/sticker" value="/res/json/sticker.json"/>
  ```
  If you set the path of the JSON file to the value in metadata, sticker information of the JSON file is automatically stored when the Sticker provider application is installed.

Reply:
Fixed.